### PR TITLE
First PR to merge basic code.

### PR DESCRIPTION
**Pull request by anijit18**

### CODE REVIEW

File: .idea/Brick_Breaker.iml [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+<module type="JAVA_MODULE" version="4">
+  <component name="NewModuleRootManager" inherit-compiler-output="true">
+    <exclude-output />
+    <content url="file://$MODULE_DIR$">
+      <sourceFolder url="file://$MODULE_DIR$" isTestSource="false" />
+    </content>
+    <orderEntry type="inheritedJdk" />
+    <orderEntry type="sourceFolder" forTests="false" />
+  </component>
+</module>
```
Comment (invicta198) [inline .idea/Brick_Breaker.iml:11]: Add an empty line at the end.

File: .idea/misc.xml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="ProjectRootManager" version="2" languageLevel="JDK_19" project-jdk-name="20" project-jdk-type="JavaSDK">
+    <output url="file://$PROJECT_DIR$/out" />
+  </component>
+</project>
```
Comment (invicta198) [inline .idea/misc.xml:6]: Same here.

**Review by invicta198 — CHANGES_REQUESTED** [review]

File: .idea/Brick_Breaker.iml (hunk 1 above)
Comment (anijit18) [reply .idea/Brick_Breaker.iml:11]: Done.

File: .idea/misc.xml (hunk 2 above)
Comment (anijit18) [reply .idea/misc.xml:6]: Done.

File: Gameplay.java [hunk 3]
```diff
@@ -0,0 +1,109 @@
+import javax.swing.JPanel;
+import java.awt.*;
+import java.awt.event.ActionEvent;
+import java.awt.event.KeyEvent;
+import java.awt.event.KeyListener;
+import java.awt.event.ActionListener;
+//import java.util.Timer;
```
Comment (invicta198) [inline Gameplay.java]: remove this comment

File: Gameplay.java [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+
+        g.dispose();
+
+    }
+
+    //no use of them
+    @Override
+    public void keyTyped(KeyEvent e) {}
+    @Override
+    public void keyReleased(KeyEvent e) {}
```
Comment (invicta198) [inline Gameplay.java:61]: If these functions have no use, remove them.

**Review by invicta198 — CHANGES_REQUESTED** [review]

**Comment by invicta198** [comment]
learn about how to add function docstrings.
Example:
```
/**
   * This is the main method which makes use of addNum method.
   * @param args Unused.
   * @return Nothing.
   * @exception IOException On input error.
   * @see IOException
   */

   public static void main(String args[]) throws IOException {
      AddNum obj = new AddNum();
      int sum = obj.addNum(10, 20);

      System.out.println("Sum of 10 and 20 is :" + sum);
   }
```

File: Gameplay.java (hunk 3 above)
Comment (anijit18) [reply Gameplay.java]: Done.

File: Gameplay.java (hunk 4 above)
Comment (anijit18) [reply Gameplay.java:61]: Main class implements KeyListener, therefore cannot remove implemented functions.

**Comment by invicta198** [comment]
Keep these files in your local but remove these files from GitHub

out/production/Brick_Breaker/Gameplay.class
out/production/Brick_Breaker/Main.class

File: .idea/Brick_Breaker.iml (hunk 1 above)
Comment (invicta198) [reply .idea/Brick_Breaker.iml:11]: Not done.

File: .idea/misc.xml (hunk 2 above)
Comment (invicta198) [reply .idea/misc.xml:6]: Not done.

File: Main.java [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+    public static void main(String[] args)
+    {
+        JFrame obj = new JFrame();
+        Gameplay gamePlay = new Gameplay();
+        obj.setBounds(10,10,700,600);
+        obj.setTitle("Breakout Ball");
+        obj.setResizable(false);
+        obj.setVisible(true);
+        obj.setDefaultCloseOperation(JFrame.EXIT_ON_CLOSE);
+
```
Comment (invicta198) [inline Main.java:13]: Remove extra whiteline

**Review by invicta198 — APPROVED** [review]

**Pull request merged**